Taskfile.yml

```diff
@@ -26,6 +26,12 @@ vars:
     bacon cargo-audit cargo-insta cargo-release default-target mdbook
     mdbook-admonish mdbook-footnote mdbook-toc wasm-bindgen-cli wasm-opt
     wasm-pack
+  brew_dependencies: |
+    brew
+    hugo
+    npm
+    python3
+    elixir
```

Comment:
I'm wondering if it's worth including elixir here.
I don't think prql-elixir has been substantially edited in the last few months.